src/skycoin/config.go

```diff
@@ -259,9 +266,13 @@ func NewNodeConfig(mode string, node NodeParameters) NodeConfig {
 		WebInterfaceCert:  "",
 		WebInterfaceKey:   "",
 		WebInterfaceHTTPS: false,
-		EnabledAPISets:    strings.Join([]string{api.EndpointsRead, api.EndpointsTransaction}, ","),
-		DisabledAPISets:   "",
-		EnableAllAPISets:  false,
+		EnabledAPISets: strings.Join([]string{
+			api.EndpointsRead,
+			api.EndpointsTransaction,
+			api.EndpointsStorage,
```

Comment:
`EndpointsStorage` shouldn't be in the default set (applies to all config modes, see https://github.com/skycoin/skycoin/#configuration-modes), since it is intended for client only.